index.html

```diff
@@ -400,25 +361,25 @@ <h3>SuspensionList2021Status</h3>
 The <code>statusListCredential</code> property MUST be a URL to a
 <a>verifiable credential</a>. When the URL is dereferenced, the resulting
 <a>verifiable credential</a> MUST have <code>type</code> property that
-includes the <code>StatusList2021Credential</code> value.
+includes the <code>StatusListCredential2021</code> value.
```

Comment:
Here and throughout:

```suggestion
includes the <code>StatusList2021Credential</code> value.
```

Reply:
Same comment as https://github.com/w3c-ccg/vc-status-list-2021/pull/24/files#r825341914.

Reply:
Fixed in 01a681de2d2c2b91fc13d5ed6b272ea27254abe4.